### DISCUSSION TITLE
Open Overseer task panel after run command

### DISCUSSION THREAD

**Discussion by scippio** [stevearc/overseer.nvim#288] [Q&A]
How can I open Overseer panel when I select (or run) command (in command selection)? Thank you!

**Comment by scippio** [comment]
I mean the task list on the right side.

**Comment by Liksium** [comment]
Maybe it will help
https://github.com/patricorgi/dotfiles/blob/main/.config/nvim/lua/overseer/component/task_list_on_start.lua

**Comment by philkoch** [comment]
I used [this response from the issues](https://github.com/stevearc/overseer.nvim/issues/36#issuecomment-1238715487) as a blueprint and created a function that I use for the keymaps instead of `run_template()`

./lua/utils.lua
```lua
local M = {}
function M.run_template_open_view(opts)
	-- opens the task view when a template is run
	-- see https://github.com/stevearc/overseer.nvim/issues/36#issuecomment-1238715487
	local overseer = require("overseer")
	overseer.run_template(opts, function(task)
		if task then
			overseer.open({ enter = false })
		end
	end)
end

return M
```

With LazyVim I defined keymaps like this:

```lua
...
keys = {
    {
        "<leader>rj",
        function()
            require("utils").run_template_open_view({ name = "just qa", params = { args = {} } })
        end,
        desc = "󰙨   Just QA-recipe",
    },
...
```
